src/main/java/hw_10/Person.java

```diff
@@ -0,0 +1,56 @@
+package hw_10;
+
+public class Person {
+    private String name;
+    private String surname;
+    private int age;
+    private int salary;
+    public Person (String name, String surname, int age, int salary){
+        this.name = name;
+        this.surname = surname;
+        this.age = age;
+        this.salary = salary;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getSurname() {
+        return surname;
+    }
+
+    public void setSurname(String surname) {
+        this.surname = surname;
+    }
+
+    public int getAge() {
+        return age;
+    }
+
+    public void setAge(int age) {
+        this.age = age;
+    }
+
+    public int getSalary() {
+        return salary;
+    }
+
+    public void setSalary(int salary) {
+        this.salary = salary;
+    }
+
+    @Override
+    public String toString() {
+        return "Person{" +
+                "name='" + name + '\'' +
+                ", surname='" + surname + '\'' +
+                ", age=" + age +
+                ", salary=" + salary +
+                '}';
+    }
```

Comment:
Please remove unused methods in Person class

Reply:
fixed

Reply:
not fixed